client/src/omtLinkProvider.ts

```diff
@@ -0,0 +1,102 @@
+
+import * as path from 'path';
+import { DocumentLink, DocumentLinkProvider, TextDocument } from 'vscode';
+import { Position, Range, Uri, workspace } from 'vscode';
+import { readFileSync } from 'fs';
+
+const MATCHER = /( {4}["']?)(.*\.omt)/;
```

Comment:
I don't think there is a need to use 4 spaces. Though it is best practice, you could use any number of spaces you like.
```suggestion
const MATCHER = /( +["']?)(.*\.omt)/;
```